apps/docs/components/hooks/useTrapFocus.md

```diff
@@ -65,11 +66,16 @@ useTrapFocus(focusTrapElementRef)
   </div>
 </template>
 ```
+
 </SourceCode>
 <!-- end vue -->
 ::::::
 
-::: slot api
+:::::: slot api
+
+::: warning DEPRECATION
+Parameter `arrowKeysOn` will be deprecated since version 2.3
+:::
```

Comment:
```suggestion
<!-- TODO: remove arrowKeysOn before 3.0.0 release -->
::: warning DEPRECATION
Parameter `arrowKeysOn` will be deprecated since version 2.3 and removed in next major version.
:::
```